.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
hmm, i'd rather not, IIUC this selects Node v=10, right? if so, i don't want to do it, as v=11 works just as fine

Reply:
It's just that I was using 12 locally

Reply:
changed to 11 and works OK